sigstore/_utils.py

```diff
@@ -139,6 +141,26 @@ def key_id(key: PublicKey) -> KeyID:
 
     return KeyID(hashlib.sha256(public_bytes).digest())
 
+def hazmat_digest_to_bundle(algo: str) -> HashAlgorithm:
```

Comment:
naive attempt to fix the conformance tests without running them locally. Still complaining that the enum number is invalid